.zuul.d/jobs.yaml

```diff
@@ -0,0 +1,14 @@
+- job:
+    name: pypa-pip-base
+    abstract: true
+    description: Run pypa/pip unit testing
+    pre-run: .zuul.d/playbooks/tox/pre.yaml
+    run: .zuul.d/playbooks/tox/main.yaml
+    post-run: .zuul.d/playbooks/tox/post.yaml
+
+- job:
+    name: pypa-pip-py38
+    parent: pypa-pip-base
+    nodeset: ubuntu-focal
+    vars:
+      tox_envlist: py38
```

Comment:
Should we look into adding more envs? Other distros?

Reply:
I feel like that was the initial reasoning for exploring Zuul :)  @ssbarnea I think was most involved in that disucssion

Reply:
Yes, I just wasn't sure if it's expected that they'd appear in this PR or follow-ups

Reply:
My preference is probably to not overload this initial support with a bunch more things, but I'm open to suggestion.  Note that any new jobs proposed in a pull request are speculatively tested by Zuul, so things will never get into a failure state

Reply:
I'm not sure what you mean by saying that things won't get into a failure state. I think I may be unfamiliar with what speculative testing actually means in this context. Mind explaining?

Reply:
What I mean is just like this change, any pull requests that modify/add/remove jobs in .zuul.d will be tested by Zuul with that config applied.  i.e. if we add a pull request to add centos/debian/XYZ jobs as a follow-up pull-request, those jobs will run on that pull request before it is committed, and we can iterate, debug, etc. That's what I mean by speculative testing; the proposed config changes are tested by Zuul from the pull request, without being committed.

Reply:
Oh, so it's basically the same as what other CIs do... I didn't know there's a special term for this behavior.